src/harness/fake_sas.py

```diff
@@ -319,11 +319,7 @@ def do_POST(self):
     elif self.path == 'admin/get_daily_activities_status':
       response = FakeSasAdmin().GetDailyActivitiesStatus()
     elif self.path == '/admin/query/propagation_and_antenna_model':
-      try:
-          response = FakeSasAdmin().QueryPropagationAndAntennaModel(request)
-      except ValueError:
-          self.send_response(400)
-          return 	  
+      response = FakeSasAdmin().QueryPropagationAndAntennaModel(request)
```

Comment:
Don't you still need the try/except block?

Reply:
An invalid configuration which will generate the exception captured here is already done in test_WINNF_FT_S_PAT_1. Thus no need here.
The PR has been updated. 
Thanks

Reply:
It seems safer to leave it in, but I agree with what you say. Since fake SAS is best-effort anyway, it's not a big deal either way.